Views/Home/Resources.cshtml

```diff
@@ -29,8 +34,8 @@
         </a>
     </li>
     <li>
-        <a href="xmpp:codingteam@conference.jabber.ru?join">
-            <i class="fa-li fa fa-lightbulb-o"></i> XMPP conference
+        <a href="https://telegram.me/joinchat/BE1AdEAiwR85TsuRlhU_bA">
+            <i class="fa-li fa fa-envelope-o"></i> Telegram group
```

Comment:
@Minoru ... and it's not extremely easy to switch to the latest font-awesome version: we need to register on their CDN or upload the font file to our server. I'll file an issue.